builtins/amp-img.js

```diff
@@ -191,10 +191,10 @@ export class AmpImg extends BaseElement {
     // fallback to stop from nested fallback abuse.
     this.allowImgLoadFallback_ = !this.element.hasAttribute('fallback');
 
-    // For inabox SSR, image will have been written directly to DOM so no need
-    // to recreate.  Calling appendChild again will have no effect.
+    // For SSR, image will have been written directly to DOM so no need to recreate.
     if (this.element.hasAttribute('i-amphtml-ssr')) {
       this.img_ = scopedQuerySelector(this.element, '> img:not([placeholder])');
+      return this.img_;
```

Comment:
The SSR can be out of date (and the attributes we currently SSR isn't synchronized with the JS here). If the only LCP issue is reattaching the node, maybe we should just skip the `appendChild` below.

Reply:
That's a great point, I'll tweak the change.

Reply:
Addressed in #782e4cd.